ios/Integreat/AppDelegate.m

```diff
@@ -33,15 +33,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
       options.APIKey = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_API_KEY;
       options.bundleID = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_BUNDLE_ID;
       options.clientID = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_CLIENT_ID;
-      // options.trackingID = ??
       options.projectID = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_PROJECT_ID;
       options.androidClientID = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_REVERSED_CLIENT_ID;
       options.databaseURL = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_DATABASE_URL;
-      // options.deepLinkURLScheme = ??
       options.storageBucket = @BUILD_CONFIG_IOS_GOOGLE_SERVICES_STORAGE_BUCKET;
+      
+      // The following values are unset as they are probably not relevant for Messsaging:
+      // options.deepLinkURLScheme = ??
+      // options.trackingID = ??
       // options.appGroupID = ??
       
-      // Unused info from GoogleService-Info.plist
+      // Unused info from GoogleService-Info.plist:
+      // These exist in the .plist file but I was not able to add them to the FIROptions. They are probably not important.
```

Comment:
```suggestion
      // These existed in the .plist file but I was not able to add them to the FIROptions. They are probably not important.
```

Also: Couldn't we remove them from the build config then?

Reply:
I would leave them here as it is unclear what effects this has. The documentation is not that good.